CHANGELOG.md

```diff
@@ -6742,3 +6742,4 @@
 [@arika]: https://github.com/arika
 [@soroktree]: https://github.com/soroktree
 [@alexevanczuk]: https://github.com/alexevanczuk
+[@such]: https://github.com/such
```

Comment:
Can you remove it? Because it is automatically generated by release task.

Reply:
Done!